.eslintrc.json

```diff
@@ -384,7 +384,8 @@
         ],
         "@typescript-eslint/no-this-alias": "error",
         "@typescript-eslint/no-unused-vars": "off",
-        "@typescript-eslint/no-var-requires": "off"
+        "@typescript-eslint/no-var-requires": "off",
+        "@typescript-eslint/switch-exhaustiveness-check": "error"
```

Comment:
You probably don't want to actually turn this on as part of this PR, as it will have effects beyond the scope of what you are changing. I also ran into some issues with trying to enable it, so I'd say hold off on this for now and investigate further in a separate change.

Reply:
True, I should have made the change in a separate pr. My bad.